src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -1,29 +1,96 @@
-import { useI18n } from '@wordpress/react-i18n';
-import Button from 'src/components/button';
-import Modal from 'src/components/modal';
+import { Guide } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { cx } from 'src/lib/cx';
+import { getIpcApi } from 'src/lib/get-ipc-api';
+import previewSitesIllustration from '../assets/preview-sites-illustration.svg';
+import versionSwitchIllustration from '../assets/version-switch-illustration.svg';
```

Comment:
I think we could consider using absolute paths for imports:

```suggestion
import previewSitesIllustration from 'src/modules/whats-new/assets/preview-sites-illustration.svg';
import versionSwitchIllustration from 'src/modules/whats-new/assets/version-switch-illustration.svg';
```

Reply:
Updated in https://github.com/Automattic/studio/pull/1128/commits/84b352825fdb7a745f20ac76a97626ad02225096